sdk/spring/spring-cloud-azure-appconfiguration-config/pom.xml

```diff
@@ -63,12 +63,12 @@
 		<dependency>
 			<groupId>com.azure.spring</groupId>
 			<artifactId>spring-cloud-azure-service</artifactId>
-			<version>4.6.0</version> <!-- {x-version-update;com.azure.spring:spring-cloud-azure-service;dependency} -->
+			<version>4.7.0</version> <!-- {x-version-update;com.azure.spring:spring-cloud-azure-service;dependency} -->
```

Comment:
Hi, @mrm9084 , in my understanding, we should use `current` instead of `dependency` here.